EIPS/eip-6956.md

```diff
@@ -0,0 +1,515 @@
+
+### ASSET-BOUND NON-FUNGIBLE TOKENS
+
+In this standard we propose to
+
+1. Elevate the concept of representing physical or digital off-chain `ASSETS` by on-chain anchoring the `ASSET` inseperably into an NFT.
+1. Being off-chain in control over the `ASSET` must mean being on-chain in control over the anchored NFT.
+1. (Related) A change in off-chain ownership over the `ASSET` inevitably should be reflected by a change in on-chain ownership over the anchored NFT.
+
+As 2. and 3. indicate, the control/ownership/posession of the ASSET should be the source of truth, *not* the posession of an NFT anchored the ASSET. Hence, we propose an `ASSET-BOUND NFT`, where off-chain CONTROL over the ASSET enforces on-chain CONTROL over the anchored NFT.
+Also the proposed ASSET-BOUND NFTs allow to anchor digital metadata inseperably to the `ASSET`. When the `ASSET` is a physical asset, this allows to design "phygitals" in their purest form, i.e. creating a "phygital" asset with a physical and digital component that are inseperable. [Note that metadata itself can still change, e.g. for "Evolvable NFT"]
+
+We propose to complement the existing transfer control mechanisms of a token according to ERC-721, `Approval` according to [ERC-721](eip-721.md) and `Permit` according to [ERC-4494](eip-4494.md), by another mechanism; `ATTESTATION`. An ATTESTATION is signed off-chain by the ORACLE and must only be issued when the ORACLE verified that whoever specifies the `to` address or beneficiary address has simultanously been in control over the ASSET. The `to` address of an attestation may be used for Transfers as well as for approvals and other authorizations.
+
+Transactions authorized via `ATTESTATION` shall not require signature or approval from neither the `from` (donor, owner, sender) nor `to` (beneficiary, receiver) account, i.e. making transfers permissionless. Ideally, transaction are signed independent from the `ORACLE` as well, allowing different scenarios in terms of gas-fees.
+
+Lastly we want to mention two major side-benefits of using the proposed standard, which drastically lowers hurdles in onboarding web2 users and increase their security;
+
+- New users, e.g `0xaa...aa` (Fig.1), can use gasless wallets, hence participate in Web3/dApps/DeFi and mint+transfer tokens without ever owning crypto currency. Gas-fees may be paid through a third-party account `0x..gasPayer` (Fig.1). The gas is typically covered by the ASSET issuer, who signs `transferAnchor()` transactions
+- Users cannot get scammed. Common attacks (e.g. wallet-drainer scams) are no longer possible or easily reverted, since only the anchored NFT can be stolen, not the ASSET itself. Also mishaps like transferring the NFT to the wrong account, losing access to an account etc can be mitigated by executing another `transferAnchor()` transaction based on proofing control over the `ASSET`, i.e. the physical object.
```

Comment:
Q: is this putting physical asset thief in capacity to change the NFT property to their benefit?

Reply:
Yes - it really maps the physical world 1:1, including the not so nice things. So you need to protect the physical asset as you'd do with any other physical thing you own. 

That being said, there is flexibility. As outlined, the ERC-6956 can easily be extended with a lockable-mechanism, which locks attestation transfers. This is important for e.g. DeFi, where the asset is collateral, but could also be used for theft protection. Since the proposed standard is completely compatible with ERC-721, any "standard lock mechanism for NFTs" can be used. If you just want to lock attestations (e.g. when using IERC6956Floatable), you can implement logic in the `_beforeAttestationUse` hook as suggested in the Reference Implementation.

Reply:
Added this question to *Security Considerations* in https://github.com/ethereum/EIPs/commit/9d188578511cb53c00ebe0910043ab50fb40a248 - Thank you!